recipes/scsplit/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "scSplit" %}
+{% set version = "1.0.8.2" %}
+
+package:
+  name: scsplit
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scSplit-{{ version }}.tar.gz
+  sha256: e39e2e840ec34bec289619817d7ccee70165e79a361041b1b38e53beac1b7fa3
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('scsplit', max_pin="x.x.x.x") }}
```

Comment:
Does this tool follow a semantic version? 
See https://bioconda.github.io/contributor/linting.html#missing-run-exports and https://semver.org/

Reply:
Thank you very much for the comments, I add the corrections to the recipe